### DISCUSSION TITLE
Facing Issue In Installation Superset(2.1.0)) for Creating Visualization Plugins

### DISCUSSION THREAD

**Discussion by roxxrroy** [apache/superset#24721] [Q&A / Help]
I am doing these step to first clone,install and run the superset code repo in my local system -

git clone https://github.com/apache/superset.git  
cd superset
git checkout 2.1.0
docker-compose up

![Screenshot (4)](https://github.com/apache/superset/assets/139281982/2be390ce-2213-454e-bc53-20eec3fc0d72)

File "/app/superset/__init__.py", line 21, in <module>
superset_worker        |     from superset.app import create_app
superset_worker        |   File "/app/superset/app.py", line 23, in <module>
superset_worker        |     from superset.initialization import SupersetAppInitializer
superset_worker        |   File "/app/superset/initialization/__init__.py", line 33, in <module>
superset_worker        |     from superset.extensions import (
superset_worker        |   File "/app/superset/extensions/__init__.py", line 32, in <module>
superset_worker        |     from superset.utils.async_query_manager import AsyncQueryManager
superset_worker        |   File "/app/superset/utils/async_query_manager.py", line 26, in <module>
superset_worker        |     from superset.utils.core import get_user_id
superset_worker        |   File "/app/superset/utils/core.py", line 72, in <module>
superset_worker        |     import bleach
superset_worker        | ModuleNotFoundError: No module named 'bleach'

I am getting this error for last 2 days, which i am not able to solve, please help me installing and setting up the code in for doing local changes

**Comment by sebastianliebscher** [comment]
Hi @roxxrroy, in order to run Superset 2.1.0 you would also need to run docker-compose with the `TAG` variable:

- `TAG=2.1.0 docker-compose up`
